### PR TITLE
chore: Add README, CONTRIBUTING and basic open source setup files to the repo

### PR DESCRIPTION
**Pull request by nkshah2**
## Summary
- Updates README
- Adds CONTRIBUTING
- Removes end-to-end from mocharc (it would warn when running tests)
- Adds an init script

## Remaining TODOs
- [ ] ...

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+    ```
+    ./startTestingEnv --wait
+    ```
+
+3. Open a new terminal in `supertokens-web-js` and run all tests
+
+    ```
+    npm run test
+    ```
```
Comment (rishabhpoddar) [inline CONTRIBUTING.md:61]: I mean.. this isn't accurate. The way we test this SDK is via auth-react SDK. So it should talk about setting auth-react SDK + core and then linking this SDK in auth react and then running tests for auth-react.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,32 @@
 # supertokens-web-js (In progress)
+
 SuperTokens SDK for vanilla JS for all recipes
+
```
Comment (rishabhpoddar) [inline README.md:4]: Should all be supertokens.com, not supertokens.io in this. I am referring to all the links

File: README.md [hunk 3]
```diff
@@ -1,2 +1,32 @@
...
+    alt="chat on Discord"></a>
+    
+## About
+This is a plain Javascript SDK that provides the login functionality with SuperTokens.
+
+Learn more at https://supertokens.io
+
+## Documentation
+
+To see documentation, please click [here](https://supertokens.io/docs/community/introduction).
```
Comment (rishabhpoddar) [inline README.md:20]: Should link to the guides page.

File: init.sh [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# This script is used to initialise the dev env for this package.
```
Comment (rishabhpoddar) [inline init.sh:1]: Why do we need this file? This was in auth-react cause there it was failing with node 16 if we only did `npm i -d`. Does this fail here too?

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (nkshah2) [reply CONTRIBUTING.md:61]: So this is true to run the few unit tests (storage abstraction and querier). When we add the pipeline of testing with auth-react I will be mentioning the steps for that here too

File: init.sh (hunk 4 above)
Comment (nkshah2) [reply init.sh]: We dont, removed the script file and the package.json script

File: CONTRIBUTING.md (hunk 1 above)
Comment (rishabhpoddar) [reply CONTRIBUTING.md:61]: So then i can't merge this PR until then.

**Pull request merged**